_episodes/05-loop.md

```diff
@@ -630,7 +630,7 @@ $ for datafile in NENE*A.txt NENE*B.txt; do echo $datafile stats-$datafile; done
 ~~~
 {: .language-bash}
 
-Using the left arrow key,
+Using the up arrow key,
```

Comment:
She's already pressed the up arrow key. At this point, she's making edits, using the left arrow key.

Reply:
Thanks. Updated it, tried to clarify.